modules/st2flow-details/style.css

```diff
@@ -385,3 +385,6 @@ limitations under the License.
 
   }
 }
+/* &-custom-toggler{
+  padding:0;
+} */
```

Comment:
Please delete commented out code and ensure there is a newline at the bottom of the file.